pkg/traceql/engine_metrics.go

```diff
@@ -1175,12 +1250,21 @@ func (b *SimpleAggregator) Combine(in []*tempopb.TimeSeries) {
 			}
 
 			b.ss[ts.PromLabels] = existing
+			if b.initAvg {
```

Comment:
Struggling a bit with how to include the average in the Combiner. I think this way we only allocate the extra bytes only for this case